runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPointsOverlay.java

```diff
@@ -64,22 +65,43 @@ public Dimension render(Graphics2D graphics)
 		int totalPoints = client.getVar(Varbits.TOTAL_POINTS);
 		int personalPoints = client.getVar(Varbits.PERSONAL_POINTS);
 
+
 		panel.getChildren().clear();
 		panel.getChildren().add(LineComponent.builder()
-			.left("Total:")
-			.right(POINTS_FORMAT.format(totalPoints))
-			.build());
+				.left("Total:")
```

Comment:
Is there a reason you added extra indentation to these?

Reply:
No

Reply:
Strangely i dont see the indentation when i look at the whole file, or in Intellij